src/popup/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -287,10 +286,16 @@ export default {
     return pick(this.$store.state.observables, ['balance', 'balanceCurrency']);
   },
   computed: {
-    ...mapGetters(['formatCurrency', 'formatNumber', 'accounts']),
+    ...mapGetters(['formatCurrency', 'formatNumber', 'accounts', 'account']),
     ...mapState('accounts', ['activeIdx']),
     ...mapState('fungibleTokens', ['aePublicData', 'availableTokens']),
     ...mapGetters('fungibleTokens', ['tokenBalances']),
+    getSimplexLink() {
+      const link = new URL(SIMPLEX_URL);
+      link.searchParams.set('wallet_address', this.account.address);
+      return link.toString();
+    },
```

Comment:
I think that method should be extracted somewhere else to not repeat it every time we want to use it